src/main/java/com/slimebot/commands/WordchainCommand.java

```diff
@@ -0,0 +1,66 @@
+package com.slimebot.commands;
+
+import com.slimebot.games.Game;
+import com.slimebot.games.PlayerGameState;
+import com.slimebot.games.games.Wordchain;
+import de.mineking.discord.commands.annotated.ApplicationCommand;
+import de.mineking.discord.commands.annotated.ApplicationCommandMethod;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+import net.dv8tion.jda.api.interactions.components.selections.StringSelectMenu;
+
+import java.time.Instant;
+
+
+@ApplicationCommand(name = "wordchain", description = "Spiele das Wordchain minispiel mit anderen Leuten!")
+public class WordchainCommand extends ListenerAdapter {
+    @ApplicationCommandMethod
+    public void performCommand(SlashCommandInteractionEvent event) {
+        Long memberId = event.getMember().getIdLong();
+        if(PlayerGameState.isInGame(memberId)) {
+            event.reply("Du bist schon in einem Game!").setEphemeral(true).queue();
+            return;
+        }
+
+        Game game = new Wordchain(memberId, event.getChannel(), event.getGuild().getIdLong());
+
+        EmbedBuilder embedBuilder = new EmbedBuilder()
+                .setTitle(event.getMember().getEffectiveName() + "'s Wortkette: Einstellungen")
+                .addField(":timer: Timeout", "Ändere die Zeit die man hat, um sich ein Wort zu überlegen", false)
+                .addField(":x: Fehler Punkte", "Ändert die Anzahl der Fehler, die gemacht werden dürfen", false)
+                .setTimestamp(Instant.now());
+
+        event.replyEmbeds(embedBuilder.build())
+                .addActionRow(
```

Comment:
You might want to use DiscordUtils' `ui` package for and make every setting a frame. Instead of a start-button you can just start the game in the handler of the 'lives' selectmenu.